administrator/components/com_templates/tmpl/style/edit.php

```diff
@@ -45,17 +47,31 @@
                     </span>
                 </div>
                 <div>
-                    <p><?php echo Text::_($this->item->xml->description); ?></p>
                     <?php
-                    $this->fieldset = 'description';
-                    $description = LayoutHelper::render('joomla.edit.fieldset', $this);
+                        $this->fieldset    = 'description';
+                        $short_description = Text::_($this->item->xml->description);
+                        $long_description  = LayoutHelper::render('joomla.edit.fieldset', $this);
+
+                        if (!$long_description) {
+                            $truncated = HTMLHelper::_('string.truncate', $short_description, 550, true, false);
+
+                            if (strlen($truncated) > 500) {
+                                $long_description  = $short_description;
+                                $short_description = HTMLHelper::_('string.truncate', $truncated, 250);
+
+                                if ($short_description == $long_description) {
+                                    $long_description = '';
+                                }
+                            }
+                        }
```

Comment:
```suggestion
                    $this->fieldset    = 'description';
                    $short_description = Text::_($this->item->xml->description);
                    $long_description  = LayoutHelper::render('joomla.edit.fieldset', $this);

                    if (!$long_description) {
                        $truncated = HTMLHelper::_('string.truncate', $short_description, 550, true, false);

                        if (strlen($truncated) > 500) {
                            $long_description  = $short_description;
                            $short_description = HTMLHelper::_('string.truncate', $truncated, 250);

                            if ($short_description == $long_description) {
                                $long_description = '';
                            }
                        }
                    }
```
Fix PHPCS failing in Drone due to too much indentation.